web/themes/custom/pop/pop.theme

```diff
@@ -163,4 +163,20 @@ function pop_form_alter(&$form, $form_state) {
   if($form['#form_id'] == 'search_block_form') {
     $form['keys']['#attributes']['placeholder'][] = t('Search');
   }
+}
+
+use Drupal\views\ViewExecutable;
+use Drupal\views\Plugin\views\query\QueryPluginBase;
+
+/**
+ * Implements hook_views_query_alter().
+ */
+function pop_views_query_alter(ViewExecutable $view, QueryPluginBase $query) {
+
+  // homepage_featured view
+  if ($view->id() == 'homepage_featured') {
+    echo 'View accessed.';
+  }
+
+  echo 'View accessed.';
```

Comment:
Pass the user facign text inside a `t()` function. Eg: `t('View accessed');`

Change the same at other places as well.

Reply:
Made suggested changes. Thanks.